draft-ietf-mls-protocol.md

```diff
@@ -2217,7 +2220,9 @@ The client verifies the validity of a LeafNode using the following steps:
   * If instead the LeafNode appears in a message being received by the client, e.g.,
     a proposal, a commit, or a ratchet tree of the group the client is joining, it is
     RECOMMENDED that the client verifies that the current time is within the range
-    of the `lifetime` field.
+    of the `lifetime` field.  (This check is not required because the LeafNode
```

Comment:
```suggestion
    of the `lifetime` field.  (This check is not compulsory because the LeafNode
```

Reply:
"compulsory" is poorly understood. Please leave as "required"

Reply:
I can see the confusion here if you read "not required" as "not needed" as opposed to "not an absolute requirement".  Suggest "not mandatory", since this is well understood to have the same semantic as "required" (see, e.g., "mandatory to implement").

```suggestion
    of the `lifetime` field.  (This check is not mandatory because the LeafNode
```